src/bitmap/backend/atomic_bitmap_arc.rs

```diff
@@ -0,0 +1,81 @@
+use std::ops::Deref;
```

Comment:
Please add license and copyright header.

Reply:
done